lib/elixir/lib/exception.ex

```diff
@@ -157,6 +157,160 @@ defmodule Exception do
   end
 
   @doc """
+  Attaches information to exceptions for extra debugging.
+
+  This operation is potentially expensive, as it reads data
+  from the filesystem, parse beam files, evaluates code and
+  so on. Currently the following exceptions may be annotated:
+
+    * `FunctionClauseError` - annotated with the arguments
+      used on the call and available clauses
+
+  """
+  @spec blame(:error, any, stacktrace) :: t
+  @spec blame(kind, payload, stacktrace) :: payload when payload: var
```

Comment:
The first argument has overlapping domain. Also the return is `{t, stacktrace} | {payload, stacktrace}`.